packages/cubejs-snowflake-driver/src/SnowflakeDriver.ts

```diff
@@ -450,7 +453,9 @@ export class SnowflakeDriver extends BaseDriver implements DriverInterface {
 
       await this.execute(connection, 'ALTER SESSION SET TIMEZONE = \'UTC\'', [], false);
       await this.execute(connection, `ALTER SESSION SET STATEMENT_TIMEOUT_IN_SECONDS = ${this.config.executionTimeout}`, [], false);
-
+      if (this.ignoreCase) {
```

Comment:
I realized that we mess the value of our env `CUBEJS_DB_SNOWFLAKE_QUOTED_IDENTIFIERS_IGNORE_CASE` / `this.ignoreCase` and the value we pass to the Snowflake.

I mean — if we have `true` — we pass `false` and vice versa.
Maybe it would be better to align them. Possible options imho are:
1. Rename our var to smth like `CUBEJS_DB_SNOWFLAKE_QUOTED_IDENTIFIERS_RESPECT_CASE`. Then having it set to `true` — means exactly what we set by this statement — not ignoring the case.
2. Or.... Use `CUBEJS_DB_SNOWFLAKE_QUOTED_IDENTIFIERS_IGNORE_CASE` as is: if it is true — then — we should pass `true` to Snowflake. In this case, don't forget to update the default value if var is not set, as we want respect case by default.